benchmark/buffers/buffer-read.js

```diff
@@ -16,21 +16,18 @@ const types = [
   'Int16BE',
   'Int32LE',
   'Int32BE',
-  'FloatLE',
-  'FloatBE',
-  'DoubleLE',
-  'DoubleBE',
 ];
 
 const bench = common.createBenchmark(main, {
-  buffer: ['fast', 'slow'],
+  buffer: ['fast'],
   type: types,
   n: [1e6]
 });
```

Comment:
We already test the float and double functions in a different benchmark as well. Therefore these seem somewhat redundant.

The same about slow and fast buffer as already explained.